sdk/tables/data-tables/review/data-tables.api.md

```diff
@@ -18,8 +20,44 @@ export interface AccessPolicy {
     start: Date;
 }
 
+// @public
+export interface AccountSASOptions {
+    expiresOn?: Date;
+    ipRange?: SasIPRange;
+    permissions?: AccountSASPermissions;
+    protocol?: SASProtocol;
+    resourceTypes?: string;
+    startsOn?: Date;
+    version?: string;
+}
+
+// @public
+export class AccountSASPermissions {
```

Comment:
It feels like a lot of effort to make a helper class just to convert the thing to/from a string.

What about two helper functions like `accountSasPermissionsToString(permissions: AccountSASPermissionsLike): string` and `stringToAccountSasPermissions(permissionString: string): AccountSASPermissionsLike` ?